### ISSUE TITLE
Export to CSV may not work with multiple query providers

### ISSUE THREAD

**Issue by Enet4** [bioinformatics-ua/dicoogle#250]
I have discovered this bug while fiddling with #244. Basically, an export to CSV file operation on the server might only send one of the query provider's search results. This happens because each query task is dispatched in parallel, and all appear to be writing to the same temporary file in the server. One of the tasks will be overwriting another task's entries, which in my case led to a table with no entries (one of my private databases would yield no results, whereas lucene would have given me the right entries).
